### PR TITLE
replication automation feature

### PR DESCRIPTION
**Pull request by leenaj0hn**

### CODE REVIEW

**Comment by shaik-gousa-khaja-asif** [comment]
Resolve conflicts

File: tests/integration/hpe_3par_api_remoteCopy_test.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+           7. Start container and append data.
+        '''
+        client = docker.from_env(version=TEST_API_VERSION) 
+        volume_name = helpers.random_name()
+        container_name= helpers.random_name()
+        self.tmp_volumes.append(volume_name)
+
+        # Create volume with -o replicationGroup option.
+        volume = self.hpe_create_volume(volume_name, driver=HPE3PAR, replicationGroup='testRCG1')
+        import pdb; pdb.set_trace();
```
Comment (shaik-gousa-khaja-asif) [inline tests/integration/hpe_3par_api_remoteCopy_test.py]: please remove this

File: tests/integration/hpe_3par_manager.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 CLIENT_CERT = cfg['etcd']['client_cert']
 CLIENT_KEY = cfg['etcd']['client_key']
 HPE3PAR_API_URL = cfg['backend']['3Par_api_url']
+HPEREMOTE3PAR_API_URL = cfg['replication_device']['hpe3par_api_url']
```
Comment (shaik-gousa-khaja-asif) [inline tests/integration/hpe_3par_manager.py]: Please commit this change as well

**Comment by prablr79** [comment]
@leenaj0hn why this changes still in dangling state, why not merged ?

File: tests/integration/hpe_3par_api_remoteCopy_test.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        targetrcgName = self.hpe_recover_remote_copy_group('testRCG1', 7)
+
+        container.start()
+
+
+        #Create contianer, mount volume and write data to file
+        container.exec_run("sh -c 'echo \"beautiful\" >> /data/test'")
+        ExecResult = container.exec_run("cat /data/test")
+        self.assertEqual(ExecResult.exit_code, 0)
+        container.stop()
```
Comment (shaik-gousa-khaja-asif) [inline tests/integration/hpe_3par_api_remoteCopy_test.py]: Assert for the data written

**Comment by prablr79** [comment]
guys when this will be merged ? still conflicting ?

File: tests/integration/hpe_3par_api_remoteCopy_test.py (hunk 1 above)
Comment (leenaj0hn) [reply tests/integration/hpe_3par_api_remoteCopy_test.py]: Removed pdb

File: tests/integration/hpe_3par_manager.py (hunk 2 above)
Comment (leenaj0hn) [reply tests/integration/hpe_3par_manager.py]: Added above line

File: tests/integration/hpe_3par_api_remoteCopy_test.py (hunk 3 above)
Comment (leenaj0hn) [reply tests/integration/hpe_3par_api_remoteCopy_test.py]: Assert included

**Pull request merged**